src/qt/clientmodel.cpp

```diff
@@ -277,7 +265,11 @@ void ClientModel::subscribeToCoreSignals()
             qDebug() << "NotifyAlertChanged";
             Q_EMIT alertsChanged(getStatusBarWarnings());
         });
-    m_handler_banned_list_changed = m_node.handleBannedListChanged(std::bind(BannedListChanged, this));
+    m_handler_banned_list_changed = m_node.handleBannedListChanged(
+        [this]() {
+            qDebug() << QString("%1: Requesting update for peer banlist").arg(__func__);
+            banTableModel->refresh();
```

Comment:
dd924e3efa021dc2fe50da9fedba4a87ce3026aa

I think this is wrong because `BanTableModel::refresh()` needs to be called on the Qt main thread.

Maybe:
```cpp
    GUIUtil::ObjectInvoke(banTableModel, [=] {
        banTableModel->refresh();
    });
```

Reply:
Thanks! [Fixed](https://github.com/bitcoin-core/gui/pull/581#issuecomment-1100709598).